ui-hedayat-suggetion/frontend/src/index copy.ts

```diff
@@ -0,0 +1,14 @@
+import "./view/header/page-header";
```

Comment:
Do you want to include this “index copy” file?

Reply:
no no , we need to delete this,
thanks for review.